### DISCUSSION TITLE
Feedback for “Azure OpenAI”

### DISCUSSION THREAD

**Discussion by laxas** [LibreChat-AI/docs#144] [Q&A]
I get the follwing error:

LibreChat         | 2024-10-18 09:48:15 error: Config file YAML format is invalid: ENOENT: no such file or directory, open '/app/librechat.yaml'

The librechat.yaml file is in the same folder (top folder) as the .env. 

I am not sure if it can't find the config or if there is a specific problem in the config. 

The yaml itself is correct (https://www.librechat.ai/toolkit/yaml_checker).

Setting a absolute CONFIG_PATH in the .env did not help.

**Comment by fuegovic** [comment]
see: https://www.librechat.ai/docs/quick_start/custom_endpoints#step-1-create-or-edit-a-docker-override-file
